src/engine/Sim3D.ts

```diff
@@ -452,7 +453,13 @@ export class Sim3D extends EventEmitter {
     this.cameraManager = new CameraManager();
 
     // Debug
-    const emptyGeometry = new THREE.Geometry();
+    const emptyGeometry = new THREE.BufferGeometry();
```

Comment:
I checked the related discourse thread and https://discourse.threejs.org/t/three-geometry-will-be-removed-from-core-with-r125/22401 moving to `BufferGeometry` seems like the right move 👍🏻

Reply:
Didn't spot the threeJS discourse! nice resource yeah using Geometry always looked a little funky to me too.